SustainabilityOnMars/GuidedNotebooks/pets-in-space/challenge-1.ipynb

```diff
@@ -0,0 +1,148 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <p style=\"text-align: center;\"> 📌 Challenge 1: Accessing the Pet Archives </p>\n",
+    "\n",
+    "In this challenge you are tasked with using computer commands to access the Pet Archives Data which was collected by Bootstrap World. \n",
+    "\n",
+    "Here are some tips and tricks to help you out: \n",
+    "- Python is very picky about capitalization and spelling, even if you have everything else right!\n",
+    "- Make sure every open bracket `(`, `[`, `[[`, has a closing one, `]]`, `]`, `)`\n",
+    "- No one gets it right the first time, so make sure you have fun!\n",
+    "\n",
+    "**Rover**: *ruff!* The first thing we should do is tell our notebook that we want to access the group of computer commands that lets us work with data. I think it was called `pandas`..?\n",
+    "\n",
+    "![https://publicdomainvectors.org/en/free-clipart/Server-rack-cabinet/83360.html](images/archives.jpg)\n",
+    "\n",
+    "# <p style=\"text-align: center;\"> 🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟 </p>\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#❗️Run this cell\n",
+    "import interactive as i\n",
+    "from IPython.display import YouTubeVideo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#❗️Run this cell\n",
+    "YouTubeVideo('P6KNkb7xVjk') "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#❗️Run this cell\n",
+    "i.challenge1a()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 📕 Debrief: Functions, Libraries, and Filenames\n",
+    "Now that we have access to the special commands we need to access the data, let's use them! The data file they gave us is called `pets_from_bootstrap_world.csv` - the .csv at the end tells us it is a csv file. The CSV stands for \"Comma Separated Values\", there is a special function in the `pandas` library we can use to view this data file called `read_csv()`. \n",
+    "\n",
+    "Python is a little tricky though, and needs us to put in the code which library the function we want to use comes from. For example, if I wanted to use the `loadtxt()` function from the `numpy` library, I would have to write it like `numpy.loadtxt('file.txt')`. See how the library is first, then there is a `.`, followed by the function? We then use the function on the file by putting the filename in the brackets. The file name is surrounded by `\"` `\"` so that it is read as characters. \n"
```

Comment:
Python is a little tricky though, and needs us to write the code to tell it which library the function we want to use comes from.